.github/workflows/build.yml

```diff
@@ -88,6 +88,94 @@ jobs:
         run: ctest -VV --output-on-failure -C ${{ matrix.buildType }}
         working-directory: _build
 
+
+# A build job matrix based on pre-built USD binaries provided by NVIDIA on Windows.
+  nvidia-usd-binaries-windows-build:
+    strategy:
+      matrix:
+        usdVersion:
+          - 20.08
+          - 20.11
+          - 21.02
+        include:
+          - usdVersion: 20.08
+            usdVersionUrl: 20.08
+            pythonVersion: 3.6
+            USE_PYTHON_3: ON
+            buildType: Release
+            buildTests: ON
+          - usdVersion: 20.11
+            usdVersionUrl: 20-11
+            pythonVersion: 3.6
+            USE_PYTHON_3: ON
+            buildType: Release
+            buildTests: ON
+          - usdVersion: 21.02
+            usdVersionUrl: 21-02
+            pythonVersion: 3.6
+            USE_PYTHON_3: ON
+            buildType: Release
+            buildTests: ON
+    runs-on: windows-2016
+    name: 'Windows 2016 NVIDIA Pre-built Binaries
+      <USD Version=${{ matrix.usdVersion }},
+       Python Version=${{ matrix.pythonVersion }},
+       Build type:${{ matrix.buildType }},
+       Enable tests=${{ matrix.build-tests }}>'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.pythonVersion }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.pythonVersion }}
+      - name: Download and extract pre-built USD binaries
+        run: |
+          Invoke-WebRequest https://developer.nvidia.com/usd-${{ matrix.usdVersionUrl }}-binary-windows-python-${{ matrix.pythonVersion }} -OutFile $env:TEMP/usd-${{ matrix.usdVersion }}.zip
+          mkdir -Force $env:TEMP/usd-${{ matrix.usdVersion }}
+          7z x $env:TEMP/usd-${{ matrix.usdVersion }}.zip $("-o" + "$env:TEMP" + "\usd-${{ matrix.usdVersion }}") -y
+      - name: Create build directories
+        run: |
+          mkdir -Force _build
+          mkdir -Force _install
+
+      - name: Run CMake
+        run: |
+          $env:BUILD_ROOT="$env:GITHUB_WORKSPACE\_build"
+          $env:USD_ROOT="$env:TEMP\usd-${{ matrix.usdVersion }}"
+          $env:PATH= "$env:BUILD_ROOT\lib;" + "$env:USD_ROOT\bin;" + "$env:USD_ROOT\lib;" + "$env:PATH"
+          $env:PYTHONPATH = "$env:BUILD_ROOT\lib\python;" + "$env:USD_ROOT\lib\python;" + "$env:PYTHONPATH"
+          $env:PXR_PLUGINPATH_NAME = "$env:BUILD_ROOT\lib\usd;" + "$env:BUILD_ROOT\plugin\usd;" + "$env:PXR_PLUGINPATH_NAME"
```

Comment:
Hey @colevfx - could you describe why the `PATH`, `PYTHONPATH`, and `PXR_PLUGINPATH_NAME` environment variables are set up here?

I thought that the previous work you did around set_test_properties already sets up these variables for the test runtime environment?  Unless this is for something else? :-)

Reply:
You're right! I changed my testing methodology locally where I removed all environment references to USD, and noticed errors popping up if I didn't add these. But this is less than ideal--I'll re-audit our `_usd_set_test_properties` function, I probably just missed something last time. 

Related to this, I didn't quite appreciate that ctest doesn't seem to update based on your local environment. Like the environment I have at time of configuration seems locked in time as far as the build and testing is concerned. So, if I change my environment, I have to re-run cmake before being able to test the changes in ctest. Is that normal? Seems like it would make integration testing a lot slower, since I'd have to re-compile the program (and others like it) every time anything environmentally changes and I need to re-run tests. 😄